uefi-raw/src/protocol/ata.rs

```diff
@@ -143,7 +145,7 @@ pub struct AtaPassThruProtocol {
         this: *const Self,
         port: u16,
         port_multiplier_port: u16,
-        device_path: *mut *mut DevicePathProtocol,
+        device_path: *mut *const DevicePathProtocol,
```

Comment:
```suggestion
```

Can you explain, please? In the [spec](https://uefi.org/specs/UEFI/2.10/13_Protocols_Media_Access.html#efi-ata-pass-thru-protocol-getnextport) we can find this definition

```C
typedef
EFI_STATUS
(EFIAPI *EFI_ATA_PASS_THRU_GET_NEXT_DEVICE) (
  IN EFI_ATA_PASS_THRU_PROTOCOL    *This,
  IN UINT16                        Port,
  IN OUT UINT16                    *PortMultiplierPort
  );
```

there is no device_path field and the port_multiplier_port field has a wrong type?

Reply:
Also it looks as BuildDevicePath() is missing between get_next_device and get_device

Reply:
Oh indeed, I was fooled by GitHubs diff view. Thanks!